pages/linux/ethtool.md

```diff
@@ -3,6 +3,22 @@
 > Display and modify Network Interface Controller (NIC) parameters.
 > More information: <http://man7.org/linux/man-pages/man8/ethtool.8.html>.
 
+- Print the current settings for an interface:
+
+`ethtool {{eth0}}`
+
+- Print the driver information for an interface:
+
+`ethtool --driver {{eth0}}`
+
+- Print the network usage statistics for an interface:
+
+`ethtool --statistics {{eth0}}`
+
+- Blink one or more LEDs on an interface for 10 seconds:
```

Comment:
I always this this it's cool that you can do this kind of stuff via the CLI (software, in general) 🚀 😄

Reply:
Unfortunately, it's working on fewer and fewer network cards. But when you do need it and it does work, it's a real time saver!